packages/x-types/src/result/result.model.ts

```diff
@@ -36,4 +36,4 @@ export interface Result extends NamedModel<'Result'>, Identifiable, Taggable {
  *
  * @public
  */
-export interface ResultVariant extends Omit<Result, 'id' | 'modelName' | 'tagging'> {}
+export interface ResultVariant extends Omit<Result, 'id' | 'modelName' | 'tagging' | 'variants'> {}
```

Comment:
Avoid infinite loop regresion when using it on AiOverviewSuggestionsSearchResponse